python/tests/integration/arcticdb/test_arctic.py

```diff
@@ -260,6 +260,32 @@ def test_staged_data(arctic_library, finalize_method):
     assert sym_without_metadata_vit.metadata is None
 
 
+@pytest.mark.parametrize("finalize_method", (StagedDataFinalizeMethod.APPEND, StagedDataFinalizeMethod.WRITE))
+@pytest.mark.parametrize("validate_index", (True, False, None))
+def test_parallel_writes_and_appends_index_validation(arctic_library, finalize_method, validate_index):
+    lib = arctic_library
+    sym = "test_parallel_writes_and_appends_index_validation"
+    if finalize_method == StagedDataFinalizeMethod.APPEND:
+        df_0 = pd.DataFrame({"col": [1, 2]}, index=[pd.Timestamp("2024-01-01"), pd.Timestamp("2024-01-02")])
+        lib.write(sym, df_0)
+    df_1 = pd.DataFrame({"col": [3, 4]}, index=[pd.Timestamp("2024-01-03"), pd.Timestamp("2024-01-04")])
+    df_2 = pd.DataFrame({"col": [5, 6]}, index=[pd.Timestamp("2024-01-03T12"), pd.Timestamp("2024-01-05")])
+    lib.write(sym, df_2, staged=True)
+    lib.write(sym, df_1, staged=True)
+    if validate_index is None:
+        # Test default behaviour when arg isn't provided
+        with pytest.raises(SortingException):
```

Comment:
Also need tests for cases where the validation should _pass_

Reply:
Agree, I'll add it to `test_parralel.py` though since the v2 API just passes through anyway